### PR TITLE
Send broken integration emails on broken calendar.

### LINKED ISSUE #8197: [CAL-1458] Send Broken Integration Email when Calendar is broken
Currently if calendar is broken, meeting gets booked in Cal.com and no communication is sent through email.

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-1458](https://linear.app/calcom/issue/CAL-1458/send-broken-integration-email-when-calendar-is-broken)</sub>

### PR DESCRIPTION
**Pull request by hariombalhara**
## What does this PR do?

See https://threads.com/34461148315?s=1QYhgXtfsUaQ5ufUDeWm4x
Fixes #8197

Fixes
- No booking confirmation email when there is a broken calendar connection.
**Environment**:  Production

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- Chore (refactoring code, technical debt, workflow improvements)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

## How should this be tested?

- [ ] Test A
- [ ] Test B

## Checklist

- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

File: packages/core/CalendarManager.ts [hunk 1]
```diff
@@ -253,9 +254,7 @@ export const createEvent = async (
           calError = error.calError;
         }
         log.error("createEvent failed", error, calEvent);
-        // @TODO: This code will be off till we can investigate an error with it
-        //https://github.com/calcom/cal.com/issues/3949
```
Comment (hariombalhara) [inline packages/core/CalendarManager.ts:257]: We should identify if the broken integration email is still sent for successful calendar connection as well.

Comment (joeauyeung) [reply packages/core/CalendarManager.ts:257]: So the reason we disabled this was it was sending false negatives on production but we couldn't replicate the issue locally. I would be hesitant.

Comment (joeauyeung) [reply packages/core/CalendarManager.ts:257]: I wouldn't be opposed to this idea.

Comment (zomars) [reply packages/core/CalendarManager.ts:257]: Agreed!

Comment (hariombalhara) [reply packages/core/CalendarManager.ts:257]: Reopening this PR and we need to handle this as well https://github.com/calcom/cal.com/pull/11702

**Comment by hariombalhara** [comment]
closing it. Should be reopened, once ready.

**Pull request closed without merging**